centos6 failing llfuse install breaks installation for tests

llfuse installation breaks the tests (py34, same for py35):

```
==> centos6_32: py34 installdeps: -rrequirements.d/development.txt, -rrequirements.d/attic.txt, -rrequirements.d/fuse.txt
==> centos6_32: ERROR: invocation failed (exit code 1), logfile: /vagrant/borg/borg/.tox/py34/log/py34-1.log
==> centos6_32: ERROR: actionid: py34
==> centos6_32: msg: getenv
==> centos6_32: cmdargs: ['/vagrant/borg/borg/.tox/py34/bin/pip', 'install', '-rrequirements.d/development.txt', '-rrequirements.d/attic.txt', '-rrequirements.d/fuse.txt']
==> centos6_32: env: {'USER': 'vagrant', 'HISTCONTROL': 'ignoredups', 'PYTHONHASHSEED': '1521877074', 'SELINUX_ROLE_REQUESTED': '', 'PYENV_VIRTUALENV_INIT': '1', 'LD_LIBRARY_PATH': '/usr/lib/libfakeroot:/usr/lib64/libfakeroot:/usr/lib32/libfakeroot', 'TERM': 'vt100', 'SSH_CLIENT': '10.0.2.2 60012 22', 'PYTHON_CONFIGURE_OPTS': '--enable-shared', 'HISTSIZE': '1000', 'SHELL': '/bin/bash', 'FAKED_MODE': 'unknown-is-real', 'FAKEROOTKEY': '37139', 'LOGNAME': 'vagrant', 'VIRTUAL_ENV': '/vagrant/borg/borg/.tox/py34', 'PYENV_SHELL': 'bash', 'PATH': '/vagrant/borg/borg/.tox/py34/bin:/home/vagrant/.pyenv/versions/3.5.2/envs/borg-env/bin:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims:/home/vagrant/.pyenv/shims:/home/vagrant/.pyenv/bin:/vagrant/borg:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/vagrant/bin:/home/vagrant/bin', 'HOME': '/home/vagrant', 'PWD': '/vagrant/borg/borg', 'LD_PRELOAD': 'libfakeroot-tcp.so', 'SELINUX_LEVEL_REQUESTED': '', 'HOSTNAME': 'centos6-32', 'LANG': 'en_US.UTF-8', 'MAIL': '/var/spool/mail/vagrant', '_': '/home/vagrant/.pyenv/versions/3.5.2/envs/borg-env/bin/tox', 'LESSOPEN': '||/usr/bin/lesspipe.sh %s', 'SELINUX_USE_CURRENT_RANGE': '', 'SSH_CONNECTION': '10.0.2.2 60012 10.0.2.15 22', 'G_BROKEN_FILENAMES': '1', 'SHLVL': '4'}
==> centos6_32: 
==> centos6_32: The directory '/home/vagrant/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
==> centos6_32: The directory '/home/vagrant/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
==> centos6_32: Collecting virtualenv<14.0 (from -r requirements.d/development.txt (line 1))
==> centos6_32:   Downloading virtualenv-13.1.2-py2.py3-none-any.whl (1.7MB)
==> centos6_32: Collecting tox (from -r requirements.d/development.txt (line 2))
==> centos6_32:   Downloading tox-2.4.1-py2.py3-none-any.whl (41kB)
==> centos6_32: Collecting pytest (from -r requirements.d/development.txt (line 3))
==> centos6_32:   Downloading pytest-3.0.3-py2.py3-none-any.whl (169kB)
==> centos6_32: Collecting pytest-cov (from -r requirements.d/development.txt (line 4))
==> centos6_32:   Downloading pytest_cov-2.4.0-py2.py3-none-any.whl
==> centos6_32: Collecting pytest-benchmark (from -r requirements.d/development.txt (line 5))
==> centos6_32:   Downloading pytest_benchmark-3.0.0-py2.py3-none-any.whl
==> centos6_32: Collecting Cython (from -r requirements.d/development.txt (line 6))
==> centos6_32:   Downloading Cython-0.25.1.tar.gz (1.7MB)
==> centos6_32: Collecting attic (from -r requirements.d/attic.txt (line 1))
==> centos6_32:   Downloading Attic-0.16.tar.gz (232kB)
==> centos6_32: Collecting llfuse<2.0 (from -r requirements.d/fuse.txt (line 1))
==> centos6_32:   Downloading llfuse-1.1.1.tar.bz2 (327kB)
==> centos6_32: Collecting pluggy<0.5.0,>=0.3.0 (from tox->-r requirements.d/development.txt (line 2))
==> centos6_32:   Downloading pluggy-0.4.0-py2.py3-none-any.whl
==> centos6_32: Collecting py>=1.4.17 (from tox->-r requirements.d/development.txt (line 2))
==> centos6_32:   Downloading py-1.4.31-py2.py3-none-any.whl (81kB)
==> centos6_32: Collecting coverage>=3.7.1 (from pytest-cov->-r requirements.d/development.txt (line 4))
==> centos6_32:   Downloading coverage-4.2.tar.gz (359kB)
==> centos6_32: Collecting msgpack-python (from attic->-r requirements.d/attic.txt (line 1))
==> centos6_32:   Downloading msgpack-python-0.4.8.tar.gz (113kB)
==> centos6_32: Installing collected packages: virtualenv, pluggy, py, tox, pytest, coverage, pytest-cov, pytest-benchmark, Cython, msgpack-python, attic, llfuse
==> centos6_32:   Running setup.py install for coverage
==> centos6_32:   Running setup.py install for Cython
==> centos6_32:   Running setup.py install for msgpack-python
==> centos6_32:   Running setup.py install for attic
==> centos6_32:   Running setup.py install for llfuse
==> centos6_32:     Complete output from command /vagrant/borg/borg/.tox/py34/bin/python3.4 -c "import setuptools, tokenize;__file__='/tmp/pip-build-pp21axxy/llfuse/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-fzsn8ieg-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py34/include/site/python3.4/llfuse:
==> centos6_32:     running install
==> centos6_32:     running build
==> centos6_32:     running build_ext
==> centos6_32:     building 'llfuse' extension
==> centos6_32:     creating build
==> centos6_32:     creating build/temp.linux-i686-3.4
==> centos6_32:     creating build/temp.linux-i686-3.4/src
==> centos6_32:     gcc -pthread -Werror=declaration-after-statement -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/vagrant/.pyenv/versions/3.4.0/include/python3.4m -c src/llfuse.c -o build/temp.linux-i686-3.4/src/llfuse.o -D_FILE_OFFSET_BITS=64 -I/usr/include/fuse -DFUSE_USE_VERSION=29 -Wall -Wextra -Wconversion -Wsign-compare -DLLFUSE_VERSION="1.1.1" -Wno-unused-function -Werror=conversion -Werror=sign-compare -DHAVE_STRUCT_STAT_ST_ATIM
==> centos6_32:     In file included from src/llfuse.c:250:
==> centos6_32:     src/llfuse.h:39:2: error: #error FUSE version too old, 2.9.0 or newer required
==> centos6_32:     src/llfuse.c:1287: warning: ‘struct fuse_forget_data’ declared inside parameter list
==> centos6_32:     src/llfuse.c:1287: warning: its scope is only this definition or declaration, which is probably not what you want
==> centos6_32:     src/llfuse.c:1301: warning: ‘struct fuse_bufvec’ declared inside parameter list
==> centos6_32:     src/llfuse.c:1325: warning: ‘struct fuse_bufvec’ declared inside parameter list
==> centos6_32:     src/llfuse.c:8146: warning: ‘struct fuse_forget_data’ declared inside parameter list
==> centos6_32:     src/llfuse.c:8146: error: conflicting types for ‘__pyx_f_6llfuse_fuse_forget_multi’
==> centos6_32:     src/llfuse.c:1287: note: previous declaration of ‘__pyx_f_6llfuse_fuse_forget_multi’ was here
==> centos6_32:     src/llfuse.c: In function ‘__pyx_f_6llfuse_fuse_forget_multi’:
==> centos6_32:     src/llfuse.c:8218: error: invalid use of undefined type ‘struct fuse_forget_data’
==> centos6_32:     src/llfuse.c:8218: error: dereferencing pointer to incomplete type
==> centos6_32:     src/llfuse.c:8220: error: invalid use of undefined type ‘struct fuse_forget_data’
==> centos6_32:     src/llfuse.c:8220: error: dereferencing pointer to incomplete type
==> centos6_32:     src/llfuse.c: At top level:
==> centos6_32:     src/llfuse.c:15763: warning: ‘struct fuse_bufvec’ declared inside parameter list
==> centos6_32:     src/llfuse.c:15763: error: conflicting types for ‘__pyx_f_6llfuse_fuse_write_buf’
==> centos6_32:     src/llfuse.c:1301: note: previous declaration of ‘__pyx_f_6llfuse_fuse_write_buf’ was here
==> centos6_32:     src/llfuse.c: In function ‘__pyx_f_6llfuse_fuse_write_buf’:
==> centos6_32:     src/llfuse.c:15828: warning: passing argument 1 of ‘__pyx_f_6llfuse_PyBytes_from_bufvec’ from incompatible pointer type
==> centos6_32:     src/llfuse.c:1325: note: expected ‘struct fuse_bufvec *’ but argument is of type ‘struct fuse_bufvec *’
==> centos6_32:     src/llfuse.c: In function ‘__pyx_f_6llfuse_init_fuse_ops’:
==> centos6_32:     src/llfuse.c:25716: error: ‘struct fuse_lowlevel_ops’ has no member named ‘forget_multi’
==> centos6_32:     src/llfuse.c:25725: error: ‘struct fuse_lowlevel_ops’ has no member named ‘write_buf’
==> centos6_32:     src/llfuse.c: At top level:
==> centos6_32:     src/llfuse.c:32198: warning: ‘struct fuse_bufvec’ declared inside parameter list
==> centos6_32:     src/llfuse.c:32198: error: conflicting types for ‘__pyx_f_6llfuse_PyBytes_from_bufvec’
==> centos6_32:     src/llfuse.c:1325: note: previous declaration of ‘__pyx_f_6llfuse_PyBytes_from_bufvec’ was here
==> centos6_32:     src/llfuse.c: In function ‘__pyx_f_6llfuse_PyBytes_from_bufvec’:
==> centos6_32:     src/llfuse.c:32199: error: storage size of ‘__pyx_v_dst’ isn’t known
==> centos6_32:     src/llfuse.c:32218: warning: implicit declaration of function ‘fuse_buf_size’
==> centos6_32:     src/llfuse.c:32218: error: dereferencing pointer to incomplete type
==> centos6_32:     src/llfuse.c:32325: warning: implicit declaration of function ‘fuse_buf_copy’
==> centos6_32:     src/llfuse.c:32199: warning: unused variable ‘__pyx_v_dst’
==> centos6_32:     src/llfuse.c: In function ‘__pyx_f_6llfuse_session_loop’:
==> centos6_32:     src/llfuse.c:36200: error: storage size of ‘__pyx_v_buf’ isn’t known
==> centos6_32:     src/llfuse.c:36295: warning: implicit declaration of function ‘fuse_session_receive_buf’
==> centos6_32:     src/llfuse.c:36452: warning: implicit declaration of function ‘fuse_session_process_buf’
==> centos6_32:     src/llfuse.c:36200: warning: unused variable ‘__pyx_v_buf’
==> centos6_32:     src/llfuse.c: In function ‘__pyx_pf_6llfuse_22notify_store’:
==> centos6_32:     src/llfuse.c:38838: error: storage size of ‘__pyx_v_bufvec’ isn’t known
==> centos6_32:     src/llfuse.c:38906: error: invalid use of undefined type ‘struct fuse_buf’
==> centos6_32:     src/llfuse.c:38906: error: dereferencing pointer to incomplete type
==> centos6_32:     src/llfuse.c:38916: error: invalid use of undefined type ‘struct fuse_buf’
==> centos6_32:     src/llfuse.c:38916: error: dereferencing pointer to incomplete type
==> centos6_32:     src/llfuse.c:38925: error: invalid use of undefined type ‘struct fuse_buf’
==> centos6_32:     src/llfuse.c:38925: error: dereferencing pointer to incomplete type
==> centos6_32:     src/llfuse.c:38968: warning: implicit declaration of function ‘fuse_lowlevel_notify_store’
==> centos6_32:     src/llfuse.c:38838: warning: unused variable ‘__pyx_v_bufvec’
==> centos6_32:     /vagrant/borg/borg/.tox/py34/lib/python3.4/site-packages/Cython/Distutils/old_build_ext.py:30: UserWarning: Cython.Distutils.old_build_ext does not properly handle dependencies and is deprecated.
==> centos6_32:       "Cython.Distutils.old_build_ext does not properly handle dependencies "
==> centos6_32:     error: command 'gcc' failed with exit status 1
==> centos6_32:     
==> centos6_32:     ----------------------------------------
==> centos6_32: Command "/vagrant/borg/borg/.tox/py34/bin/python3.4 -c "import setuptools, tokenize;__file__='/tmp/pip-build-pp21axxy/llfuse/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-fzsn8ieg-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg/.tox/py34/include/site/python3.4/llfuse" failed with error code 1 in /tmp/pip-build-pp21axxy/llfuse
==> centos6_32: You are using pip version 7.1.2, however version 8.1.2 is available.
==> centos6_32: You should consider upgrading via the 'pip install --upgrade pip' command.
==> centos6_32: 
==> centos6_32: ERROR: could not install deps [-rrequirements.d/development.txt, -rrequirements.d/attic.txt, -rrequirements.d/fuse.txt]; v = InvocationError('/vagrant/borg/borg/.tox/py34/bin/pip install -rrequirements.d/development.txt -rrequirements.d/attic.txt -rrequirements.d/fuse.txt (see /vagrant/borg/borg/.tox/py34/log/py34-1.log)', 1)
```

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -40,8 +40,8 @@ def packages_debianoid
   EOF
 end
 
-def packages_redhatted
-  return <<-EOF
+def packages_redhatted(version)
+  script = <<-EOF
     yum install -y epel-release
     yum update -y
     # for building borgbackup and dependencies:
@@ -58,6 +58,13 @@ def packages_redhatted
     #pip install virtualenv
     touch ~vagrant/.bash_profile ; chown vagrant ~vagrant/.bash_profile
   EOF
+  if version == "centos6"
+    script += <<-EOF
+      # avoid that breaking llfuse install breaks borgbackup install under tox:
+      sed -i.bak '/fuse.txt/d' /vagrant/borg/borg/tox.ini
+    EOF
+  end
+  return script
 end
 
 def packages_darwin
@@ -325,7 +332,7 @@ Vagrant.configure(2) do |config|
     b.vm.provider :virtualbox do |v|
       v.memory = 768
     end
-    b.vm.provision "install system packages", :type => :shell, :inline => packages_redhatted
+    b.vm.provision "install system packages", :type => :shell, :inline => packages_redhatted("centos7")
     b.vm.provision "install pyenv", :type => :shell, :privileged => false, :inline => install_pyenv("centos7_64")
     b.vm.provision "install pythons", :type => :shell, :privileged => false, :inline => install_pythons("centos7_64")
     b.vm.provision "build env", :type => :shell, :privileged => false, :inline => build_pyenv_venv("centos7_64")
@@ -335,7 +342,7 @@ Vagrant.configure(2) do |config|
 
   config.vm.define "centos6_32" do |b|
     b.vm.box = "centos6-32"
-    b.vm.provision "install system packages", :type => :shell, :inline => packages_redhatted
+    b.vm.provision "install system packages", :type => :shell, :inline => packages_redhatted("centos6")
     b.vm.provision "install pyenv", :type => :shell, :privileged => false, :inline => install_pyenv("centos6_32")
     b.vm.provision "install pythons", :type => :shell, :privileged => false, :inline => install_pythons("centos6_32")
     b.vm.provision "build env", :type => :shell, :privileged => false, :inline => build_pyenv_venv("centos6_32")
@@ -348,7 +355,7 @@ Vagrant.configure(2) do |config|
     b.vm.provider :virtualbox do |v|
       v.memory = 768
     end
-    b.vm.provision "install system packages", :type => :shell, :inline => packages_redhatted
+    b.vm.provision "install system packages", :type => :shell, :inline => packages_redhatted("centos6")
     b.vm.provision "install pyenv", :type => :shell, :privileged => false, :inline => install_pyenv("centos6_64")
     b.vm.provision "install pythons", :type => :shell, :privileged => false, :inline => install_pythons("centos6_64")
     b.vm.provision "build env", :type => :shell, :privileged => false, :inline => build_pyenv_venv("centos6_64")

